include/swift/SILOptimizer/Analysis/Analysis.h

```diff
@@ -234,6 +237,26 @@ class FunctionAnalysisBase : public SILAnalysis {
     return it.second.get();
   }
 
+  virtual void forceConstruct(SILFunction *f) override {
```

Comment:
This function isn't used in your PR, what is it for?

Reply:
Thanks, updated in the new commit. `forcePrecompute()` is called before the pass `forceVerifyAnalyses()` is called after the pass.